pytest.ini

```diff
@@ -7,3 +7,4 @@ python_files = tests.py test_*.py *_tests.py
 filterwarnings =
     ignore:.*Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated.*:DeprecationWarning
     ignore:.*Using the add method to register a processor or pattern is deprecated. Use the `register` method instead.*:DeprecationWarning
+    ignore:.*smart_text.*is deprecated in favor of smart_str:PendingDeprecationWarning
```

Comment:
When is this thrown? Can we somehow update our code to the smart_str?

Reply:
This warning is thrown inside 3-party library so we can't do anything about this

Reply:
Ok. I thought so. ☹️